### PR TITLE
Add support for channels to docker compose testnet

### PR DESCRIPTION
**Pull request by mpowaga**

### CODE REVIEW

**Comment by dincho** [comment]
@mpowaga Could you please add a line in the [docs](https://github.com/aeternity/epoch/blob/master/docs/docker.md#localnet) mentioning this new endpoint. 

I'll be also grateful if you mention the `websocket` endpoint in the docs too.

**Comment by dincho** [comment]
Thanks

File: docker/nginx-default.conf [hunk 1]
```diff
@@ -39,14 +39,14 @@ server {
...
-        proxy_connect_timeout 15s;
-        proxy_send_timeout 2h;
-        proxy_read_timeout 2h;
-        proxy_pass_request_headers on;
+    }
+
+    location /channel {
+        include cors.conf;
+        include ws.conf;
+        proxy_pass http://node1:3014;
```
Comment (lucafavatella) [inline docker/nginx-default.conf]: node1?

File: docker/nginx-default.conf [hunk 2]
```diff
@@ -65,13 +65,13 @@ server {
...
-        proxy_send_timeout 2h;
-        proxy_read_timeout 2h;
-        proxy_pass_request_headers on;
     }
-}
+
+    location /channel {
+        include cors.conf;
+        include ws.conf;
+        proxy_pass http://node1:3014;
```
Comment (lucafavatella) [inline docker/nginx-default.conf]: node1?

**Comment by mpowaga** [comment]
Good catch @lucafavatella! Fixed it.

**Pull request merged**